data/human/intro missions.txt

```diff
@@ -413,7 +413,87 @@ mission "Intro [2 Interceptor]"
+
+			label choices
+			choice
+				`	"What equipment do I need to mine?"`
+				`	"How do you deal with pirates?"`
+					goto pirates
+				`	"How can I check the value of minerals?"`
+					goto value
+				`	"I think I have enough information."`
+					goto end
+
+			`	"Well, you need a weapon that does hull damage," James says. "A Beam Laser will get the job done, but it will take a while. If you want to optimize your mining operations, however, you'll want a Mining Laser. Not only does it cut through asteroids faster, it's also more precise and preserves more minerals when the asteroid breaks apart. Some miners like to have a bit of combat flexibility by mining with Heavy Lasers, however.`
+			`	"Aside from lasers, the most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids, which increases your profitability per run. After that, a tactical scanner will tell you how far away the asteroid is and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining, extend your exploration range, and allow you to be ready for a quick escape."`
+			choice
+				`	"I can't fit all that on my ship!"`
+				`	"That all makes sense."`
+					goto more
+
+			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to be flexible. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be of more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`
+				goto more
+
+			label pirates
+			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting merchants and plundering ships.`
+				goto more
+			
+			label value
+			`	"You will learn this by experience, but if the planet has an outfitter, you can check the price there before you sell.`
+
+			label more
+			`	"Anything else you want to ask me?"`
+				goto choices
+			
+			label end
+			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while."`
```

Comment:
```suggestion
			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post."`
```
We don't tell the player where or how to sell the minerals, so we should tell them to go to the trade panel.

Reply:
How about "trading post or outfitter"?

Reply:
New Sahara doesn't have an outfitter, but I get your point. Another option may be adding a new choice dialogue up above asking James where to sell the minerals.

Reply:
```suggestion
`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post or, if there is one, the outfitter."`
```

Reply:
```suggestion
			`	"If you're willing to take me on one more journey," James says, "meet me in the spaceport bar in a little while. Don't forget to sell off your minerals at the trading post, or if there is one, the outfitter."`
```
If you *insist*. But I *insist* the comma has to go there.